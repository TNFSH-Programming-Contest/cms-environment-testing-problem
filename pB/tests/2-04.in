210622423 -66436747
408121132 971870486
