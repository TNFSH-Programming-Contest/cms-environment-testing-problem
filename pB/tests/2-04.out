618743555 905433739
